assets/src/apps/authoring/components/PropertyEditor/schemas/screen.ts

```diff
@@ -1,18 +1,169 @@
-const screenSchema = {
+import { JSONSchema7 } from 'json-schema';
+import customFieldTemplate from '../custom/CustomFieldTemplate';
+const screenSchema: JSONSchema7 = {
   type: 'object',
   properties: {
-    customCssClass: {
-      type: 'string',
-      options: { input_width: '500px' },
+    Size: {
+      type: "object",
+      title: "Dimensions",
+      properties: {
+          width: {type:'number'},
+          height: {type:'number'}
+        }
+    },
+    Position: {
+      type: "object",
+      title: "Position",
+      properties: {
+          x: {type:'number'},
+          y: {type:'number'},
+          z: {type:'number'},
+          customCssClass: {
+            title: 'Custom CSS Class',
+            type: 'string',
+          },
+        }
+    },
+    palette: {
+      type:'object',
+      properties:{
+        backgroundColor: {type:'string'},
+        borderColor: {type:'string'},
+        borderRadius: {type:'string'},
+        borderStyle: {type:'string'},
+        borderWidth: {type:'string'},
+        fillAlpha: {type:'number'},
```

Comment:
I think all the fill stuff we should leave as "legacy" and only use the "useHtmlProps". we shouldn't render that flag or any of the legacy flags in the authoring UI, instead it should just be something that once we come into the authoring tool it should auto update to the new version